src/utils/aria.ts

```diff
@@ -5,6 +5,8 @@ export const ariaLabelOrHidden = (ariaLabel?: string) =>
     ? { 'aria-label': ariaLabel, role: 'img' }
     : { 'aria-hidden': true };
 
+export type ariaLiveModes = 'assertive' | 'polite' | 'off';
```

Comment:
We should probably name types using these conventions: https://basarat.gitbook.io/typescript/styleguide#type

And should use PascalCase for these?

Reply:
So the suggested type would be after changes like this, if I'm on the same line:
`export type AriaLiveMode = 'assertive' | 'polite' | 'off';`

- Using PascalCase
- No plural in name

Reply:
Already changed it to PascalCase, but will remove the plural too.

Reply:
Should now be in order.